source/models/relationships.md

```diff
@@ -277,31 +277,37 @@ export default DS.Model.extend({
 });
 ```
 
-When a user comments on a blogPost, we need to create a relationship between the two records. We can simply set the `belongsTo` relationship in our new comment:
+Now, suppose we want to add comments to an existing blogPost. We can do this in two ways, but for both of them, we will first need a reference to our existing blogPost as given below:
```

Comment:
How about this?
"We can do this in two ways, but for both of them, we first need to look up a blog post that is already loaded in the store, using its id:"

Reply:
It's better. I made this change.